sklearn/discriminant_analysis.py

```diff
@@ -587,6 +778,32 @@ class QuadraticDiscriminantAnalysis(ClassifierMixin, BaseEstimator):
 
         .. versionadded:: 0.17
 
+    solver : string, optional
+        Solver to use, possible values:
+          - 'svd': Singular value decomposition (default).
+            Does not compute the covariance matrix, therefore this solver is
+            recommended for data with a large number of features.
+          - 'lsqr': Least squares solution, can be combined with \
+    custom covariance estimator.
+
+        .. versionadded:: 0.22
+
+    covariance_estimator : estimator, default=None
+        The covariance estimator is a BaseEstimator with a fit
+        method and a ``covariance_`` attribute like the estimators in
+        `sklearn.covariance`. It is used to estimate the covariance matrix.
+
+        if None the shrinkage parameter drives the estimate.
+
+        .. versionadded:: 0.22
+
+    standardize : bool
```

Comment:
Maybe let's call it `covariance_standardize` in the public API to be more specific. Otherwise I would assume that standardization is applied on the data in general (similarly to normalize). The private covariance related function could still call it `standardize`.

Reply:
I think it does exactly what you suggest ie standardizing the features before estimating the cov.

Reply:
For estimating the covariance yes, but I'm not sure that there no other places we use raw X independently of this standardization parameter, say the SVD solver here https://github.com/scikit-learn/scikit-learn/blob/a58c6c8bf237041337977cc15e1f2990ae776318/sklearn/discriminant_analysis.py#L381.

More generally if this is strictly equivalent to using a StandardScaler in a pipeline then I'm not sure there is much sense in adding this parameter.

Reply:
@rth I agree this parameter is indeed a bit confusing and not necessary. I'll remove it.